DALI.tex

```diff
@@ -750,6 +750,23 @@ \subsubsection{Intervals}
 \verb|MAX| child elements to describe the (minimum) lower bound and (maximum)
 upper bound of interval(s) respectively.
 
+\subsubsection{Sexigessimal Coordinates}
+Coordinate values expressed in sexigessimal form can be described using the following
+xtypes in both VOTable \xmlel{FIELD} and \xmlel{PARAM} elements:
+
+right ascension: \verb|datatype="char"| \verb|arraysize="*"| \verb|xtype="hms"|
+
+declination: \verb|datatype="char"| \verb|arraysize="*"| \verb|xtype="dms"|
+
+For xtype="hms", the values are serialised as hours:minutes:seconds where hours
+and minutes are integer values and seconds is a real value. For xtype="dms", the values
+are serialised as degrees:minutes:seconds where degrees and minutes are integer
+values and seconds is a real value. All hours must fall within [0,23], degrees
+must fall within [0,359], minutes must fall within [0,59], and seconds
+must fall within [0,59]; the minimum values are thus 0:0:0 (hms and dms) and the maximum
+values are ~23:59:59.9 (hms) and ~359:59:59.9 (dms) (decimal values may have
+additional precision).
```

Comment:
yes, dms should go from -90:0:0 to 89:59:59.9...

In other coordinates we limited the range of valid values so that providers were responsible for range reduction and I haven't heard any complaints about that, but here it's more markup for existing widespread use so I'm a lot less sure,

Reply:
I just realised that in the other coordinate validity specs we just said [-90,90] and [0,360] without worrying to much about the endpoint wrap around (which could have been [0,360) if we wanted to be really careful). Do you think the more relaxed spec like [0,360] is enough?

Reply:
Since as you say this is mostly about marking up existing content I'd incline to being more permissive here.  But we don't have [0,360], we have [-90,90], and both extrema have to be allowed in that case anyway.

Another thing: I suggest also clarifying that an optional explicit "+" sign on dms is allowed.

Reply:
optional "+" at the front, because "-" is also a possibility? and we don't need that for hms because they are only positive anyway

Reply:
added optional + for dms